lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py

```diff
@@ -106,6 +112,22 @@
           disk:
             bus: virtio
 
+- name: Create virtual machine 'myvm' with multus network interface
+  kubevirt_vm:
+      name: myvm
+      namespace: vms
+      memory: 512M
+      interfaces:
+        - name: default
+          bridge: {}
```

Comment:
Everything looks fine except this part – if it's fine to have an empty `bridge` parameter, then why don't we just not require specifying it in the config? Let's have it just default to {} and not have the user think about it.

Reply:
For now I would merge as-is and think of general solution for handling reasonable nested defaults.

Reply:
@mmazur It is ok with you?

Reply:
Sure, this is a minor issue.

Though if we have an agreed upon list of improvements, where do we put it? Maybe in the module itself as a comment?

Reply:
Or an issue which could be handled by next PR.

Reply:
https://github.com/kubevirt/ansible-kubevirt-modules/issues/137